tasks/SmoothieChef.tex

```diff
@@ -1,29 +1,91 @@
 
+\noindent\textbf{Reward:} 750pts.
 
-\subsection{Setup}
+
+\subsection*{Bonus rewards}
+\begin{enumerate}[nosep]
+	% Not intended to be addressed in 2019
+	\item Pouring \emph{milk} without spilling (500pts).
+	% Not intended to be addressed in 2019
+	\item Pouring \emph{sugar} without spilling (750pts).
+	\item Activate the blender/stir with the mixer (250pts).
+\end{enumerate}
+
+\subsection*{Setup}
 \begin{itemize}[nosep]
-	\item \textbf{Kitchen Counter:} Place plastic fruit onto the counter in the kitchen.
-	\item \textbf{Ingredient Bowl:} Place bowl into the kitchen.
+	\item \textbf{Location:} The test takes place in the kitchen.
+
+	\item \textbf{Start Location:} The robot starts outside the arena.
+	When the door is open, it navigates to the kitchen counter.
+
+	\item \textbf{Operator location:} The operator is standing in front of the robot on the opposite side of the counter.
+
+	\item \textbf{Kitchen Counter:} Either the kitchen table or counter can be used in this test.
+	All ingredients are placed in the kitchen counter near to the \emph{Official Blender}.
+	A spoon is available to add \emph{sugar} from a bowl.
+
 \end{itemize}
 
-\subsection{Referee instructions}
+\subsection*{Additional rules and remarks}
+\begin{enumerate}
+	\item \textbf{Deus ex Machina:} Score reduction is applied as follows:
+	\begin{itemize}[nosep]
+		\item \textbf{Start in place:} Starting the test in front of the kitchen counter causes a score reduction of 150pts.
+		\item \textbf{Reminding recipe:} Telling the robot which ingredient comes next (e.g.~pointing, naming, etc.) causes a score reduction of 250pts.
+		\item \textbf{Hand-over:} Handing over an object causes a score reduction of 200pts.
+	\end{itemize}
+
+	\item \textbf{Custom operator:} The training is normally conducted by a \emph{Professional Operator}. Teams can opt for a \emph{Custom Operator}, which causes a score reduction of 250pts.
+
+	\item \textbf{Partial bonus scoring:} Spilling \emph{milk} or \emph{sugar} causes 75\% score reduction on the associated bonuses (i.e.~400pts and 600pts respectively).
+
+	\item \textbf{Smoothie Ingredients:} There are 6 \emph{fruits}, one bottle of \emph{milk}, and a bowl with \emph{sugar} in the counter.
+	All of them are standard objects (see~\refsec{rule:scenario_objects}).
+	The \emph{milk} can be any liquid (e.g.~milk, water, \emph{drinks} category).
+	The \emph{sugar} can be any pourable solid (e.g.~muesli or cereal), is contained in a bowl and must be served with a spoon.
+
+	\item \textbf{Smoothie Recipe:} Three out of six random fruits to be blended with one teaspoon of \emph{sugar} and a glass of \emph{milk}, incorporated on team's request.
+
+	\item \textbf{Official Blender \& Mixer:} Unless a real blender is available (unlikely) teams can expect a glass or bowl to be used in this test. Any large object (e.g.~plastic bottle) from the list of standard objects (see~\refsec{rule:scenario_objects}) can be used as mixer.
+\end{enumerate}
```

Comment:
How do the “activate” or turn on a fake blender?

Reply:
You turn on the blender (twist knob, push button, I don't know) or stir with the mixer (or whatever it's called) which is assumed to be on.

![image](https://user-images.githubusercontent.com/8583949/53199213-c09dda00-361e-11e9-94e6-7c8046ae7cbd.png)